lib/library/html.cc

```diff
@@ -65,19 +65,38 @@ static void generate_entry(const beatmap_entry &entry, std::ostream &os)
  */
 static void generate_set(const beatmap_set &set, std::ostream &os)
 {
-	os << "<article>";
-	os << "<h4>" << escape{set.artist} << " - " << escape{set.title} << "</h4><ul>";
+	os << "<h3>" << escape{set.title} << "</h3><ul>";
 	for (const beatmap_entry &entry : set.entries)
 		generate_entry(entry, os);
-	os << "</ul></article>";
+	os << "</ul>";
```

Comment:
I feel like we should keep a semantic element to identify a beatmap set, like a `<section>` maybe.